src/CSS.cpp

```diff
@@ -13,36 +13,49 @@ namespace CSS
     {
         sProvider = gtk_css_provider_new();
 
-        struct CSSDir
+        std::vector<std::string> locations;
+        std::string home = std::getenv("HOME");
+        const char* env;
+
+        if ((env = std::getenv("XDG_CONFIG_HOME")))
+        {
+            locations.push_back(env);
+        }
+        else if (!home.empty())
+        {
+            locations.push_back(home.append("/.config"));
```

Comment:
```home + "/.config"```
std::string::append mutates the string, which we don't want (In the next fallback this adds ```~/.config/.local/share``` instead of ```~/.local/share```)